steering/CHARTER.md

```diff
@@ -81,7 +81,11 @@ and community, beginning in August 2020.
         the seat for the term.
     1. Because the goal of Community Seats is to increase the perspectives on
     the Steering Committee, employees of a Company that holds Contribution
-    Seats are ineligible to be elected to hold a Community Seat.
+    Seats are ineligible to hold a Community Seat.
```

Comment:
Do we need to mention by when the special election will be held? Additionally, since we have 1 seat less until then how does the voting/decision process work?

Reply:
We don't specify a timeframe for the regular elections, so I didn't want to put one here.
The seat is community voted, not Steering voted.

Reply:
I was referring to voting within Steering for decisions with 1 seat less (i.e. 12 seats)  if the elections and filling the seat takes longer than what we desire.

Reply:
Oh, sorry; the charter expresses all voting in terms of percentage of the Seats, and in the common case, 60% of 12 or 13 both round up to needing 8 votes. I don't see that causing any issues.

Reply:
I thought about the timing question. I started to write something, and then personally thought that boxing ourselves in was unnecessary. "Special Election" by definition means at an unusual time.

I don't think the number of seats in Steering matters, since this election is from the community. For other decisions when Steering is understaffed, the 60%/80% thresholds apply.